packages/@adobe/spectrum-css-temp/components/card/index.css

```diff
@@ -44,236 +145,1091 @@ governing permissions and limitations under the License.
+  }
+  .spectrum-Card-grid:after {
+    z-index: 2;
+  }
+  min-inline-size: var(--spectrum-card-quiet-min-size);
+
+  .spectrum-Card-grid {
+    block-size: 100%;
+    display: grid;
+    grid-template-columns: minmax(0, auto) 1fr minmax(0, auto);
+    grid-template-rows: auto var(--spectrum-card-quiet-body-header-margin-top) minmax(30px, auto) auto var(--spectrum-card-body-padding-bottom);
+    grid-template-areas:
+      "preview preview preview"
+      ".       .       ."
+      "title   detail  actionmenu"
+      "content content content"
+      ".       .       .      ";
 
-  .spectrum-Card-preview {
-    flex-shrink: 0;
-    min-height: 0;
+    &:before {
+      content: ' ';
+      grid-area: preview;
+
+      border-radius: var(--spectrum-card-quiet-border-radius);
+      transition: background-color var(--spectrum-global-animation-duration-100);
+    }
+
+    &:after {
+      content: ' ';
+      grid-area: preview;
+      position: absolute;
+      inset-inline-start: 0;
+      inset-inline-end: 0;
+      inset-block-start: 0;
+      inset-block-end: 0;
+      margin: calc(var(--spectrum-alias-focus-ring-gap) * -2);
+      padding: calc(var(--spectrum-alias-focus-ring-gap) * 2);
+      background-clip: content-box;
+
+      border-radius: calc(var(--spectrum-card-quiet-border-radius) + var(--spectrum-alias-focus-ring-gap));
+    }
+    .spectrum-Card-decoration {
+      display: block;
+      grid-area: preview;
+      inline-size: 100%;
+      block-size: 100%;
+
+      border-radius: var(--spectrum-card-quiet-border-radius);
+    }
+  }
+  .spectrum-Card-image,
+  .spectrum-Card-illustration {
     display: flex;
+
+    align-self: center;
+    justify-self: center;
+
+    border-radius: var(--spectrum-card-quiet-border-radius);
+  }
+  .spectrum-Card-image {
+  }
+  .spectrum-Card-illustration {
+    display: grid; /* not sure why flex doesn't work here */
     align-items: center;
-    justify-content: center;
+    justify-items: center;
+    inline-size: 100%;
+    block-size: var(--spectrum-card-coverphoto-height);
+  }
+  .spectrum-Card-avatar {
+    display: none;
+  }
+  .spectrum-Card-heading {
+    align-self: center;
+    margin-inline-end: var(--spectrum-card-title-padding-right);
+  }
+  .spectrum-Card-detail {
+    align-self: center;
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    margin-inline-end: var(--spectrum-card-title-padding-right);
+  }
+  .spectrum-Card-actions {
+    align-self: start;
+  }
+  .spectrum-Card-content {
+    margin-block-start: 5px;
+    margin-block-end: 5px;
+  }
+  .spectrum-Card-divider {
+    display: none;
+  }
+  .spectrum-Card-footer {
+    display: none;
+  }
+}
+/** END Quiet Vertical **/
+
+/** Standard Horizontal **/
+/** Tile **/
+/**
+Broken as a CSS only solution in FF until aspect-ratio is supported or this is resolved https://bugzilla.mozilla.org/show_bug.cgi?id=1436180
+For now working around it with a useLayoutEffect in CardBase.
+**/
```

Comment:
Looks like it is supported in 89 according to MDN. Only Safari 15 though. Is it possible to use aspect-ratio here to work around the FF bug though but not rely on it fully so old Safari works? I don't think I understand the bug.

Reply:
I tried using aspect-ratio, it didn't work in FF, at least, not in the way that chrome did
I think this bug is related that I've linked, and I've asked as much in the bug thread, so we'll see if anyone chimes in on it

Reply:
So you can try this codesandbox https://codesandbox.io/s/angry-alex-3w55s?file=/src/App.js in Chrome/ Safari/ FF
This sets aspect-ratio on our image container, and this seems to work in Chrome, and now it's working in FF but I swear it wasn't before.
It still doesn't work in Safari though. Aspect ratio gives "unsupported property value"
<img width="459" alt="Screen Shot 2021-09-02 at 11 20 27 AM" src="https://user-images.githubusercontent.com/698229/131896383-69cc2164-2df3-4f24-b1d6-8778998d4721.png">
I could pass the width/height direct to our image possibly now and dispense with the spacer img

Reply:
done